diff --git a/index.d.mts b/index.d.mts
new file mode 100644
index 0000000000000000000000000000000000000000..3fe3b3281538838f971f5dfdfca2a70002f447b4
--- /dev/null
+++ b/index.d.mts
@@ -0,0 +1 @@
+export * from './dist/index.d.ts'
\ No newline at end of file
diff --git a/worker.d.mts b/worker.d.mts
new file mode 100644
index 0000000000000000000000000000000000000000..f97219f51b09328da1e185087f52af20e291a84f
--- /dev/null
+++ b/worker.d.mts
@@ -0,0 +1 @@
+export * from "./dist/worker/index.d.ts"
